### PR TITLE
Bumping spaghetti version

### PR DESCRIPTION
**Pull request by jGaboardi**
- [x] The justification for this PR is: 

Bumping the tagged version of [`spaghetti`](https://github.com/pysal/spaghetti/releases/tag/v1.3.1). This release incorporates more detailed instructions for installation and no change in functionality.

See #1123, pysal/spaghetti#294, and pysal/spaghetti#296 for details.

### CODE REVIEW

File: tags.json [hunk 1]
```diff
@@ -1 +1 @@
-{"libpysal": "v4.1.0", "esda": "v2.1.1", "giddy": "v2.2.0", "inequality": "v1.0.0", "pointpats": "v2.1.0", "spaghetti": "v1.3", "mapclassify": "v2.1.1", "spreg": "v1.1.0", "segregation": "1.1.1", "spglm": "v1.0.7", "spint": "v1.0.5", "splot": "v1.1.1", "mgwr": "v2.1.1", "spvcm": "v0.2.1post1"}
\ No newline at end of file
+{"libpysal": "v4.1.0", "esda": "v2.1.1", "giddy": "v2.2.0", "inequality": "v1.0.0", "pointpats": "v2.1.0", "spaghetti": "v1.3.1", "mapclassify": "v2.1.1", "spreg": "v1.1.0", "segregation": "1.1.1", "spglm": "v1.0.7", "spint": "v1.0.5", "splot": "v1.1.1", "mgwr": "v2.1.1", "spvcm": "v0.2.1post1"}
```
Comment (sjsrey) [inline tags.json]: tags.json is dervied in the build script so no need to edit it here.

File: tarballs.json [hunk 2]
```diff
@@ -1 +1 @@
...
+{"libpysal": ["v4.1.0", "https://api.github.com/repos/pysal/libpysal/tarball/v4.1.0"], "esda": ["v2.1.1", "https://api.github.com/repos/pysal/esda/tarball/v2.1.1"], "giddy": ["v2.2.0", "https://api.github.com/repos/pysal/giddy/tarball/v2.2.0"], "inequality": ["v1.0.0", "https://api.github.com/repos/pysal/inequality/tarball/v1.0.0"], "pointpats": ["v2.1.0", "https://api.github.com/repos/pysal/pointpats/tarball/v2.1.0"], "spaghetti": ["v1.3.1", "https://api.github.com/repos/pysal/spaghetti/tarball/v1.3.1"], "segregation": ["1.1.1", "https://api.github.com/repos/pysal/segregation/tarball/1.1.1"], "mapclassify": ["v2.1.1", "https://api.github.com/repos/pysal/mapclassify/tarball/v2.1.1"], "splot": ["v1.1.1", "https://api.github.com/repos/pysal/splot/tarball/v1.1.1"], "spreg": ["v1.1.0", "https://api.github.com/repos/pysal/spreg/tarball/v1.1.0"], "spglm": ["v1.0.7", "https://api.github.com/repos/pysal/spglm/tarball/v1.0.7"], "spint": ["v1.0.5", "https://api.github.com/repos/pysal/spint/tarball/v1.0.5"], "mgwr": ["v2.1.1", "https://api.github.com/repos/pysal/mgwr/tarball/v2.1.1"], "spvcm": ["v0.2.1post1", "https://api.github.com/repos/pysal/spvcm/tarball/v0.2.1post1"]}
```
Comment (sjsrey) [inline tarballs.json]: tarballs.json is derived in the build script so no need to edit.

**Comment by jGaboardi** [comment]
Are we concerned about the two failures in the [Python 3.6 test](https://travis-ci.org/pysal/pysal/jobs/566476435) or should I just go ahead and merge? They both seem to be precision [errors `segregation` tests](https://travis-ci.org/pysal/pysal/jobs/566476435#L2372-L2460).

**Comment by sjsrey** [comment]
No merge, yet. Want to see the dust settle on the travis changes. Can discuss tomorrow at the dev meetings.

**Pull request merged**